zio-schema-json/shared/src/main/scala/zio/schema/codec/JsonCodec.scala

```diff
@@ -58,7 +65,7 @@ object JsonCodec {
         )
     }
 
-  implicit def schemaBasedBinaryCodec[A](implicit schema: Schema[A]): BinaryCodec[A] =
+  implicit def schemaBasedBinaryCodec[A](implicit schema: Schema[A], cfg: Config): BinaryCodec[A] =
```

Comment:
Now `Config.default` is no longer implicit, but here it is an implicit parameter which makes this a bit inconvenient. I suggest to keep an `implicit def schemaBasedBinaryCodec` with only schema parameter, always using `Config.default`, so users who want to customize it can use the explicit version.

Reply:
Ahh sorry, I missed that part